packages/svelteui-core/src/components/Blockquote/Blockquote.d.ts

```diff
@@ -0,0 +1,11 @@
+import { HTMLAttributes } from 'svelte/elements';
+import type { SvelteUIColor, SvelteUINumberSize } from '$lib/styles';
+import { TextProps } from '../Text/Text';
+
+export interface BlockquoteProps extends TextProps<HTMLHeadingElement> {
```

Comment:
We should also extend `HTMLBlockquoteAttributes` here, for the `cite` attribute

Reply:
The `cite` does not receive any props from the user, so I haven't added it to the types. Should we add `citeProps` and spread it in the `cite`? Though, I can't think of any use case for this. Being a slot, the user can inject any markup they want into it.